src/sbmlutils/test/report/test_report.py

```diff
@@ -69,3 +86,9 @@ def check_report_math_type(sbml_path: Path, math_type: str, tmp_path):
     # check the returned HTML in the variable for correctness of type
     assert html
     assert isinstance(html, str)
+
+
+@pytest.mark.parametrize("sbml_path", uncertainty_paths, ids=sbml_paths_idfn)
```

Comment:
In addition to this general tests write some specific tests using the `uncertainty_example.xml` from the examples. 
- create a copy in the test data folder.
- get the info for the uncertainties and check via asserts that the content of the dictionaries is correct.

Reply:
Similar to the comment above, I would copy the ```uncertainty_example.xml``` to the new ```testsuite``` directory and run a specific test for it. I hope this would work fine for our tests. Please let me know if I should use some other approach for making the testsuite. I would do all the required changes and re-upload for your reviews. Thanks!

Reply:
For the testsuite I meant to include the actual SBML testsuite models, but keep these separate from our own additional test models.